src/config.ts

```diff
@@ -386,6 +420,12 @@ export class ConfigurationReader implements vscode.Disposable {
     get ctestDefaultArgs(): string[] {
         return this.configData.ctestDefaultArgs;
     }
+    get packEnvironment() {
```

Comment:
See other comment, but should this be `cpackEnvironment`? I'm fine with it either way, just trying to think through what would be best.